capstone/src/main/resources/static/script.js

```diff
@@ -416,3 +441,23 @@ function closeForm() {
 	document.getElementById("organization").style.display = "none";
 }
 
+/* Function to support searching for organizations by name */
+function searchOrg() {
+  var university = sessionStorage.getItem("university");
+  if (university == null) {
+    getUserType();
+    if (sessionStorage.getItem("university") == null) {
+      return;
+    }
+  }
+  var name = document.getElementById("search-org").value;
+  fetch('search-organization?name=' + name + "&university=" + university).then(response => response.json()).then((organizations) => {
+
+    const orgListElement = document.getElementById('list-organizations');
+    orgListElement.innerHTML = '';
```

Comment:
What if 0 results are returned? Should the page say something like "No organizations found"? 
Doesn't need to be done in this PR.

Reply:
I will file an issue to address it in a later PR.